src/lib/bootstrap_controller/bootstrap_controller.ml

```diff
@@ -311,8 +311,12 @@ end = struct
                        scan state from the peer."
                     , [] ) ))
         in
-        Error.raise err
-    | Ok root_staged_ledger ->
+        Logger.info logger ~module_:__MODULE__ ~location:__LOC__
+          "Failed to find scan state from the peer or received faulty scan \
```

Comment:
Maybe include the error string with this log? I'm not sure how useful it is, but it'd be good to not lose it.

Reply:
I changed it so that error got attached to the end of the log.